network.go

```diff
@@ -174,6 +181,7 @@ func (n *network) MarshalJSON() ([]byte, error) {
 	netMap["endpointCnt"] = n.endpointCnt
 	netMap["enableIPv6"] = n.enableIPv6
 	netMap["generic"] = n.generic
+	netMap["persist"] = n.persist
```

Comment:
am thinking of the persist option as not specific to local store.
Hence for the case of `persist=true`, if we dont marshal it, it will be unmarshalled default to `persist=false` and that is incorrect.
If I remove the marshalling, then I have to hardcode the n.persist=true in Unmarshalling, which is Incorrect because, the json marshallling can be used not just for store.

Hence I decided to marshal and unmarshal it. WDYT ?